e2e/cypress/support/ldap_commands.js

```diff
@@ -0,0 +1,120 @@
+        cy.findByText('Log Out').scrollIntoView().should('be.visible').click().wait(TIMEOUTS.TINY).then(() => {
+            cy.checkLoginPage(settings);
+        });
+    });
+});
+
+Cypress.Commands.add('doInviteGuest', (user, settings = {}) => {
+    cy.checkLeftSideBar(settings);
+
+    // # Click hamburger main menu button
+    cy.get('#sidebarHeaderDropdownButton').should('be.visible').click().wait(TIMEOUTS.TINY).then(() => {
+        cy.findByText('Invite People').scrollIntoView().should('be.visible').click().wait(TIMEOUTS.TINY).then(() => {
+            cy.checkInvitePeopleAdminPage();
+
+            cy.findByText('Guests').click();
+
+            cy.findByText('Invite Guests').scrollIntoView().should('be.visible').click().wait(TIMEOUTS.TINY).then(() => {
+                // # Search and add a user
+                cy.findByTestId('emailPlaceholder').should('be.visible').within(() => {
+                    cy.get('input').type(user.username, {force: true}).wait(TIMEOUTS.TINY).type('\n');
+                });
+                cy.findByTestId('channelPlaceholder').should('be.visible').within(() => {
+                    cy.get('input').type('town', {force: true}).wait(TIMEOUTS.TINY).type('\n');
+                });
+            });
+            cy.findByText('Invite Guests').scrollIntoView().should('be.visible').click().wait(TIMEOUTS.TINY).then(() => {
+                cy.findByText('Successful Invites').scrollIntoView().should('be.visible').then(() => {
+                    cy.findByText('Done').scrollIntoView().should('be.visible').click();
+                });
+            });
+        });
+    });
+});
+
+Cypress.Commands.add('doInviteMember', (user, settings = {}) => {
+    cy.checkLeftSideBar(settings);
+
+    // # Click hamburger main menu button
+    cy.get('#sidebarHeaderDropdownButton').should('be.visible').click().wait(TIMEOUTS.TINY).then(() => {
+        cy.findByText('Invite People').scrollIntoView().should('be.visible').click().wait(TIMEOUTS.TINY).then(() => {
+            cy.checkInvitePeopleAdminPage();
+
+            cy.findByText('Members').click();
+
+            cy.findByText('Invite Members').scrollIntoView().should('be.visible').click().wait(TIMEOUTS.TINY).then(() => {
+                // # Search and add a user
+                cy.findByTestId('inputPlaceholder').should('be.visible').within(() => {
+                    cy.get('input').type(user.username, {force: true}).wait(TIMEOUTS.TINY).type('\n');
+                });
+            });
+
+            cy.findByText('Invite Members').scrollIntoView().should('be.visible').click().wait(TIMEOUTS.TINY).then(() => {
+                cy.findByText('Invite More People').scrollIntoView().should('be.visible').then(() => {
+                    cy.findByText('Done').scrollIntoView().should('be.visible').click();
+                });
+            });
+        });
+    });
+});
+
+Cypress.Commands.add('setLDAPTestSettings', (config) => {
```

Comment:
Please make this as a regular function and move in the spec file where it's being used.